java/code/src/com/redhat/rhn/common/db/datasource/xml/Package_queries.xml

```diff
@@ -751,7 +751,7 @@ ORDER BY UPPER(X.PACKAGE_NVRE)
   <elaborator name="package_channels" />
 </mode>
 
-<mode name="packages_available_to_errata_in_channel" class="com.redhat.rhn.frontend.dto.PackageOverview">
+<mode name="packages_available_to_errata_in_channel" class="com.redhat.rhn.frontend.dto.PackageOverview">, PackageOverview.class
```

Comment:
this look like a syntax error. Please check

Reply:
Fixed